docs/installation.rst

```diff
@@ -14,35 +14,78 @@ If you don't have `pip`_ installed, this `Python installation guide`_ can guide
 
     .. code-block:: console
 
-     pip install xhydro
+        conda install -c conda-forge xhydro
 
-Either of those will install the latest stable release of `xHydro` with all its dependencies, with two exception:
+This will install the latest stable release of `xHydro` with all its dependencies, with two exception:
 
 If you wish to use the `extreme_value_analysis` module, which relies on the `Extremes.jl`_ Julia package, you will need to install the `julia` extra:
 
     .. code-block:: console
 
-     pip install xhydro[julia]
+        pip install xhydro[julia]
 
 .. _Extremes.jl: https://github.com/jojal5/Extremes.jl
 
-If you wish to use the `frequency_analysis.regional` module, you will need to install the `lmoments3` library yourself. This library is available on both `PyPI` or `conda-forge`, but has a restricted license.
+If you wish to use the `frequency_analysis.regional` module, you will need to install the `lmoments3` library yourself. This library is available on both `PyPI` or `conda-forge`, but has a restricted license. Note that if you installed `xHydro` using `conda`, you will need to install `lmoments3` using `conda` as well. If you installed `xHydro` using the `environment-dev.yml` file, `lmoments3` will have been added automatically.
 
     .. code-block:: console
 
-     conda install -c conda-forge lmoments3
+        conda install -c conda-forge lmoments3
+
+Stable release (PyPI)
+---------------------
+You can also install `xHydro` from `PyPI` using `pip`:
+
+.. code-block:: console
+
+   pip install xhydro
+
+Please note that the considerations mentioned above regarding the `extreme_value_analysis` and `frequency_analysis.regional` modules also apply to the PyPI installation.
+
+Additionally, some dependencies of `xHydro` may not be fully functional when installed via `pip`. For example, packages like `xESMF` may require additional system-level configuration to work correctly.
+
+.. warning::
+
+   There is currently a known issue with the `juliacall` library when installing `xHydro` from `PyPI`. This can lead to segmentation faults when attempting to import the library.
+
+   Until this issue is resolved or a reliable workaround is identified, we recommend installing `xHydro` via `conda` if you intend to use the `extreme_value_analysis` module.
+
+Installing `ravenpy` and `raven-hydro` can be challenging in standard `pip` environments due to complex system-level dependencies. As a result, installing `xHydro` from PyPI will **not** include these two packages by default, and any related modules will be deactivated.
+
+If you wish to use Raven-based hydrological modelling, you can manually install the necessary dependencies first. On Linux or macOS, you can use the following commands:
+
+.. code-block:: console
+
+   apt-get update && apt-get upgrade -y
+   apt-get install -y git gdal-bin python3-gdal libgdal-dev gcc libnetcdf-dev
+   pip install xhydro[raven]
```

Comment:
Isn't there a meta-package called `build-essential`? We also need to have `make` and `cmake` (often installed already, but not always).

Reply:
Also, installing `python3-gdal` would only be relevant for people attempting to install `xhydro` in their system Python installation (very much not recommended).

Reply:
This is based on information from @sebastienlanglois, but feel free to suggest something else!

Reply:
If the environment is using a mixed system/virtualenv approach, this is probably fine, but if that's the case, I would be very clear about installing `xhydro` locally, not globally (`--user`).